pkg/registry/apis/provisioning/jobs/export/folders.go

```diff
@@ -38,6 +40,7 @@ func ExportFolders(ctx context.Context, repoName string, options provisioning.Ex
 	}); err != nil {
 		return fmt.Errorf("load folder tree: %w", err)
 	}
+	repositoryResources.SetTree(tree)
```

Comment:
alternatively we could return the tree and set it from the caller.

Reply:
not sure what you did this. It shouldn't be necessary. That function is ensuring the tree exists in the repository (in Github) . 

Here we read the tree in memory from unified storage and make sure it's present in Github. 


After `EnsureFolderTreeExists` runs all the elements of the tree you pass will be in the repositoryResources tree.